checkers/Game.py

```diff
@@ -13,10 +13,15 @@ def __init__(self, screen):
             0 - Selection Mode
             1 - Active Mode
         '''
-        self.mode = 0
+        self.mode = 9
```

Comment:
We can make this _mode 9_ as initial _mode 0_, and change other modes accordingly.